GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+# Jujutsu Governance
+
+## Overview
+
+Jujutsu is an open source project, led, maintained and designed for a worldwide
+community. Anyone who is interested can join, contribute, and participate in the
+decision-making process. This document is intended to help you understand how
+you can do that.
+
+## Project roles
+
+There are two broadly defined roles in the Jujutsu project: **Maintainers** and
+**Contributors**.
+
+### Maintainers
+
+At the top of the hierarchy are the **Maintainers**. We consider maintainers to
+be the people who contribute, review, guide, and collectively make decisions
+about the direction and scope of the project.
+
+A typical maintainer is not only someone who has made "large" contributions, but
+someone who has shown they are continuously committed to the project and its
+community. Some expected responsibilities of maintainers include (but are not
+exclusively limited to):
+
+- Displaying a high level of commitment to the project and its community, and
+being a role model for others.
+- Writing patches &mdash; a lot of patches, especially "glue code" or "grunt
+work" or general "housekeeping"; fixing bugs, ensuring documentation is always
+high quality, consistent UX design, improving processes, making judgments on
+dependencies, handling security vulnerabilities, and so on and so forth.
+- Reviewing code submitted by others &mdash; with an eye to maintainability,
+performance, code quality, and "style" (fitting in with the project).
+- Participating in design discussions, especially with regards to architecture
+or long-term vision.
+- When necessary, participate in a collective voting process to resolve
+conflicts and make decisions for which there is no clear answer.
+
+This is not an exhaustive list, nor is it intended that every maintainer does
+each and every one of these individual tasks to equal amounts. Rather this is
+only a guideline for what maintainers are expected to conceptually do.
+
+In short, Maintainers are the outwardly visible stewards of the project.
+
+#### Current list of maintainers
+
+The current list of maintainers:
+
+- Yuya Nishihara
+- Martin von Zweigbergk
+- Waleed Khan
+- Ilya Grigoriev
+- Austin Seipp

```

Comment:
Minor Nit: Add the respective GH/Discord usernames here, so its clear how to contact them.

Reply:
+ emails, if we think it's relevant

Reply:
and maybe sort the list?

Reply:
I think it's probably easiest for now to just put the GitHub usernames. People can just look at our profiles from there to get an email address or method of contact.